lib/features/home/bloc/location_bloc.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:equatable/equatable.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:geocoding/geocoding.dart';
```

Comment:
u already have nartus_geocoder as a service, why do u still need to use this package here?